pkgs/by-name/fr/freesweep/package.nix

```
@@ -41,7 +51,7 @@ stdenv.mkDerivation rec {
     mainProgram = "freesweep";
     homepage = "https://github.com/rwestlund/freesweep";
     license = licenses.gpl2Only;
-    maintainers = with maintainers; [ ];
+    maintainers = [ maintainers.sanana ];
```

It's fine to use `with` scoped to an array, so

```suggestion
    maintainers = with maintainers; [ sanana ];
```

or rather even

```suggestion
    maintainers = with lib.maintainers; [ sanana ];
```

if you want to get ahead of https://github.com/NixOS/nixpkgs/issues/371862